content/guides/getstarted/scaffold-nextjs-anchor.md

```diff
@@ -0,0 +1,306 @@
+target folder. IDL is a spec similar to ABI in EVM environments. It describes
+how the JSON file will interact with a solana program. copy the address inside
+metadata from your `hello_world.json` to your anchor program
+`[lib.rs](http://lib.rs)` line 3 and paste your address there.
+
+You can now run anchor tests and check if your program is running successfully.
+
+![Anchor Hello World Code Screenshot](/assets/guides/scaffolds/scaffolds---1-anchor-view.png)
+
+```
+    hello_solana
+    |
+    ├── anchor
+    │   ├── migrations
+    │   ├── programs
+		|   |   └── hello_world
+    │   └── tests
+		|   |   └── hello-world.spec.ts
+    │   └── target
+    │        └── idl
+    │            └── hello_world.json
+```
+
+To test hello_world instruction, execute `npm run anchor-test`.
+
+```
+
+ PASS   anchor  tests/hello-world.spec.ts
+  hello-world
+    ✓ Is initialized! (686 ms)
+
+Test Suites: 1 passed, 1 total
+Tests:       1 passed, 1 total
+Snapshots:   0 total
+Time:        1.207 s
+Ran all test suites.
+
+ ————————————————————————————————————————————————————————————————
+ >  NX   Successfully ran target jest for project anchor (3s)
+```
+
+### NextJS app
+
+![NextJS Starter view](/assets/guides/scaffolds/scaffolds---1-anchor-view.png)
+
+Our NextJS Scaffold is using App Router introduced in version 13, which supports
+shared layouts, nested routing, loading states, and error handling.
+
+```
+    hello_solana
+    |
+    ├── anchor
+    ├── web
+    │   ├── app
+    │   │   └── layout.tsx
+    │   ├── components
+    │       └──  solana
+		|          └── solana-provider.tsx
+    │       ├── ui
+                └── ui-layout.tsx
+```
+
+The `solana-provider.tsx` already has all the wallet features you need, It
+handles auto connects of solana wallet easily, you can move to multiple
+components of your web application with the wallet states managed. This NextJS
+app is using `[@tanstack/react-query](https://tanstack.com/query/latest)` for
+fetch, cache, synchronize, and update server state in your web applications
+easily. React-Query here is being used here for all the data fetching needs,
+like hook for `useGetBalance` to get the balance of your wallet,
+`useTransferSol` to transfer sol. Similarly, you can implement features you want
+to fetch from RPCs. You can see react-query getting used here in the Account nav
+as well.
+
+![NextJS web Accounts view](/assets/guides/scaffolds/scaffolds---3-web-accounts-view.png)
+
+This app also uses [`Jotai`](https://jotai.org), is used for global React state
+management in your NextJS app. Our component stores the cluster and rpc
+information, you can fetch it anywhere in your app and manage where the
+transaction needs to be submitted.
+
+![NextJS web Clusters view](/assets/guides/scaffolds/scaffolds---4-web-cluster-view.png)
+
+Ideally, copy your generated IDL file or Types from the target folder after the
+anchor build and import it to use inside components to call instructions.
```

Comment:
I don't understand this sentence. 

1. `Ideally` - should we do this or not? 
2. `copy your generated IDL file or Types from the target folder` - should we copy only the IDL file, or the types file, or both?

Please clarify and provide the commands to copy. 

`anchor build` should be in backticks (``)